lib/mimic/module.ex

```diff
@@ -129,7 +129,7 @@ defmodule Mimic.Module do
   if @elixir_version >= 1.18 do
     defp generate_mimic_struct(module) do
       if function_exported?(module, :__info__, 1) && module.__info__(:struct) != nil do
-        struct_info = module.__info__(:struct)
+        struct_info = module.__info__(:struct) || []
```

Comment:
Given that the `if` checks for `module.__info__(:struct) != nil` how could this `|| []` be reached? 🤔

Reply:
Actually, I found an even simpler way:
Replaced the list comprehension with an `Enum.map`. No warnings, no hacky stuff.

Reply:
Oh I see! Thanks for taking the time to explain 🙇